app/src/main/java/de/westnordost/streetcomplete/quests/address/AddHousenumberForm.kt

```diff
@@ -25,6 +25,7 @@ class AddHousenumberForm : AbstractQuestFormAnswerFragment<HousenumberAnswer>()
 
     override val otherAnswers = listOf(
         AnswerItem(R.string.quest_address_answer_no_housenumber) { onNoHouseNumber() },
+        AnswerItem(R.string.quest_address_answer_house_name_and_housenumer) { switchToHouseNameAndHouseNumber() },
         AnswerItem(R.string.quest_address_answer_house_name) { switchToHouseName() },
         AnswerItem(R.string.quest_housenumber_multiple_numbers) { showMultipleNumbersHint() }
```

Comment:
Not directly related to this PR, but we should probably standardize on whether to use lambdas or function references
```kt
AnswerItem(R.string.quest_address_answer_house_name_and_housenumer, ::switchToHouseNameAndHouseNumber)
```
Currently they are both used. I see four reasonable options:

- Prefer function references whenever possible
- Prefer lambdas for the last argument (outside the parentheses) and function references otherwise
- Prefer function references when registering callbacks and lambdas otherwise— _less sure about this one_
- Always prefer lambdas



@westnordost Any opinion?

Reply:
I slightly prefer lambdas, but I don't see the necessity that it is standardized in the codebase